src/main/java/mcmultipart/api/multipart/IMultipart.java

```diff
@@ -281,6 +281,14 @@ public default void onAdded(IPartInfo part) {
     public default void onRemoved(IPartInfo part) {
     }
 
+    public default boolean canPlacePartAt(IBlockAccess world, BlockPos pos) {
+        return true;
+    }
+
+    public default boolean canPlacePartOnSide(IBlockAccess world, BlockPos pos, EnumFacing side) {
+        return canPlacePartAt(world, pos);
```

Comment:
I wonder if this should even be proxied verbatim. I'd be in favor of passing both an EnumFacing and an IPartSlot here, not just an EnumFacing.

Reply:
Yep, could be useful in some circumstances. I wonder, why not only have canPlacePartOnSide, since it's basically the same as canPlacePartAt, but with more information?

Reply:
I'd say keep both. You don't always have more information. That, or mark the added information as optional/not-necessary-to-provide somehow.